gauge.go

```diff
@@ -103,6 +107,9 @@ func (g *Gauge) Buffer() []Point {
 				p.Bg |= AttrReverse
 			}
 
+			if g.PercentColorHighlighted != Attribute(uint16max) {
```

Comment:
Not sure if ColorClear is a good name, though. If a user sees that, he might be confused what the difference to ColorDefault is. I'd suggest that we name it something like ColorGaugeClear so that it becomes clear that the color should not be used in other widgets.